pkgs/servers/mail/vsmtp/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, openssl
+, gsasl
+, testers
+, vsmtp
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "vsmtp";
+  version = "1.1.3";
+
+  src = fetchFromGitHub {
+    owner = "viridIT";
+    repo = "vsmtp";
+    rev = "v${version}";
+    sha256 = "sha256-EXhnvr7j3MUQm1J3vS4zb9Hluy1Yr3irok4i5Wm8vwI=";
+  };
+
+  cargoSha256 = "sha256-+isT/tHEITmsDKZZIIy5piCjba18iHz/09FGehnQ4j0=";
+
+  nativeBuildInputs = [ pkg-config rustPlatform.bindgenHook ];
+  buildInputs = [ openssl gsasl ];
+
+  passthru = {
+    tests.version = testers.testVersion { package = vsmtp; };
+  };
+
+  meta = with lib; {
+    description = "A next-gen mail transfer agent (MTA) written in Rust. Faster and Greener.";
```

Comment:
Should not end with a period.

Reply:
But there are two sentences, leaving one without a period looks strange.

Reply:
What about merging them into one sentence?